klasses/views_test.py

```diff
@@ -0,0 +1,52 @@
+"""Views for bootcamps"""
+from datetime import timedelta
+
+from django.urls import reverse
+import pytest
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_403_FORBIDDEN,
+)
+
+from klasses.factories import BootcampRunFactory
+from klasses.models import BootcampRunEnrollment
+from klasses.serializers import BootcampRunSerializer
+from main.utils import now_in_utc
+
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture
+def runs(user):
+    """Make some test runs"""
+    run = BootcampRunFactory.create()
+    unavailable_run = BootcampRunFactory.create(start_date=now_in_utc()+timedelta(days=1))
+    already_enrolled_run = BootcampRunFactory.create()
+    BootcampRunEnrollment.objects.create(user=user, bootcamp_run=already_enrolled_run)
+
+    return run, unavailable_run, already_enrolled_run
```

Comment:
Please use `SimpleNamespace` or something like that instead of returning tuples (https://github.com/mitodl/bootcamp-ecommerce/blob/master/applications/serializers_test.py#L35-L38)